hedera-node/src/main/java/com/hedera/services/state/expiry/renewal/RenewalHelper.java

```diff
@@ -63,6 +64,9 @@
 	private final Supplier<FCMap<MerkleEntityId, MerkleAccount>> accounts;
 	private final Supplier<FCMap<MerkleEntityAssociation, MerkleTokenRelStatus>> tokenRels;
 
+	/* Only needed for interoperability, will be removed during refactor */
+	private final BackingAccounts backingAccounts;
+
```

Comment:
Why don't we just use `BackingStore` to avoid the type cast in `ServicesContext`?